librz/core/tui/visual.c

```diff
@@ -2038,6 +2032,7 @@ RZ_IPI void rz_core_visual_browse(RzCore *core, const char *input) {
 			rz_debug_switch_to_first_thread(core->dbg);
 			break;
 		case 'b':
+			rz_view_and_seek_to_bb(core);
 			rz_view_and_seek_to_bb(core);
```

Comment:
??? why double?

Reply:
I've fixed it already. There was some weird problem w my vscode that's why this had happened